protostar/cli/common_arguments.py

```diff
@@ -167,3 +167,10 @@
     type="str",
     default="",
 )
+
+LINKED_LIBRARIES = ProtostarArgument(
+    name="linked-libraries",
+    value_parser="list",
+    description="Libraries to include in compilation",
```

Comment:
This could use a better explanation (paths to cairo1 libraries perhaps)?

Reply:
updated